### ISSUE TITLE
columnDef visible not working with responsive

### ISSUE THREAD

**Issue by AleksandarDev** [DataTables/DataTables#692]
I have noticed that when I initialize DataTable with `responsive: true`, columnDefs marked with `visible: false` still get rendered on the table.

In this case, first column gets rendered when it shouldn't
```
"responsive": true,
"ajax": {
    "url": "<SOME URL>"
},
"columnDefs": [
	{
		"targets": 0,
		"visible": false
	}
   ...
```

**Comment by DataTables** [comment]
Hi - this will be addressed with Responsive 2.0. This feature has already been committed if you wish to try out a pre-release version. For v1.x you would need to use the [`never` class](http://datatables.net/extensions/responsive/classes) to address this.

If you do a search on the [DataTables forums](//datatables.net/forums) you will find a number of discussions about this issue.

**Comment by AleksandarDev** [comment]
@DataTables Thanks for the information. I was looking for similar issues but couldn't find any that reported this problem.

Will try the pre-release version.
